doc/setting_up_sidewalk_environment/setting_up_sidewalk_prototype.rst

```diff
@@ -86,20 +86,21 @@ The tools required for provisioning are located in the repository (`sdk-nrf`_ an
                --wireless_device_json wireless_device.json
                --device_profile_json device_profile.json
                --addr 0x17c000
+               --output_hex nordic_aws_nrf54l15.hex
 
          .. note::
             The default name of the :file:`mfg.hex` file generated from the :file:`provision.py` script is :file:`nordic_aws_nrf52840.hex`.
-            It is, however, compatible with other supported boards.
+            It is, however, compatible with other supported boards, but this name can be overwritten by parameter `--output_hex` as shown in example.
```

Comment:
If someone runs the command above then the default name won't be a problem anymore, so maybe we should rephrase the entire note to: 

```suggestion
            The ``--output_hex`` parameter is used to specify a custom name for the :file:`mfg.hex` file. 
            This change is optional and does not affect the file's compatibility with other supported boards. 
            If the parameter is not provided, the default name will be :file:`nordic_aws_nrf52840.hex`.
```

+should this be updated for the other tab as well(?)

Reply:
ok, change applied

Reply:
The change is not fully applied - can we edit the entire note as suggested?